ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-cpou-upgrade-from-4.15.yaml

```diff
@@ -0,0 +1,138 @@
+base_images:
+  ansible:
+    name: "4.15"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.17"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
+  tools:
+    name: "4.15"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  intermediate:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.16"
+  latest:
+    release:
+      architecture: amd64
+      channel: fast
+      version: "4.15"
+  target:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.17"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: ibmcloud-ipi-ovn-ipsec-f14
+  cron: 51 23 2,17 * *
+  steps:
+    cluster_profile: ibmcloud-qe
+    env:
+      FILTERS_ADDITIONAL: ~TELCO&
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.15 and @ibmcloud-ipi and not
+        @fips'
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.15 and @ibmcloud-ipi and not
+        @fips'
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE},${RELEASE_IMAGE_TARGET}" > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_INTERMEDIATE
+        name: release:intermediate
+      - env: RELEASE_IMAGE_TARGET
+        name: release:target
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - chain: openshift-upgrade-qe-test-eus
+    workflow: cucushift-installer-rehearse-ibmcloud-ipi-ovn-ipsec
+- as: nutanix-ipi-proxy-fips-mini-perm-f14
+  cluster: build01
+  cron: 2 4 2,17 * *
+  steps:
+    cluster_profile: nutanix-qe-dis
+    env:
+      FILTERS_ADDITIONAL: ~TELCO&
+      FIPS_ENABLED: "true"
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.15 and @nutanix-ipi'
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.15 and @nutanix-ipi'
+    test:
+    - as: set-upgrade-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE},${RELEASE_IMAGE_TARGET}" > ${SHARED_DIR}/upgrade-edge
+      dependencies:
+      - env: RELEASE_IMAGE_INTERMEDIATE
+        name: release:intermediate
+      - env: RELEASE_IMAGE_TARGET
+        name: release:target
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - chain: openshift-upgrade-qe-test-eus
+    workflow: cucushift-installer-rehearse-nutanix-ipi-proxy-minimal-permission
+- as: vsphere-ipi-sdn-f14
```

Comment:
`sdn` is unsupported in 4.17, and currently only chain-upgrade support snd-to-ovn migration during upgrade.

Reply:
I see sdn is still supported in 4.15, so leaving it in one profile for test coverage and also to get test result for the team to evaluate.

Reply:
> only the sdn one.

@jianlinliu about sdn one, i commented here. for cpou test currently, i guess the team's goal is to explore issues as more as possible to help identify if we should release this in 4.17. so i think there is no harm to keep it in the scope, correct me if anything i might miss.

actually sdn is also in 4.15-4.16/4.15-4.17 upgrade jobs, so i think we could leave one profile to cover sdn in 4.15-4.17 cpou, right? 
```
$ grep "\- as:" openshift-openshift-tests-private-release-4.16__amd64-nightly-4.16-upgrade-from-stable-4.15.yaml|grep sdn
- as: vsphere-ipi-sdn-f28

$ grep "\- as:" openshift-openshift-tests-private-release-4.17*4.15*|grep sdn
openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.15.yaml:- as: vsphere-ipi-sdn-f28
```

> only chain-upgrade support snd-to-ovn migration

yes, `openshift-upgrade-qe-test-eus` chain call chain-upgrade step, so it's supported in eus.

Reply:
Though the snd2ovn migration function exists in `cucushift-chainupgrade-toimage` included in the eus chain, but the sdn networking is existing in each single node, while I am not sure if the sdn2ovn take the eus upgrade into consideration (only upgrade contral plane nodes, do not touch with worker nodes), and seen from the vsphere-ipi-sdn [job](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/55226/rehearse-55226-periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-4.17-cpou-upgrade-from-4.15-vsphere-ipi-sdn-f14/1821127283583225856) , it failed on `sdn2ovn` migration function. cc @zhaozhanqi to check.

Reply:
I noticed the issue during checking the result, so I listed in the description. I assume:
if it's a flake, then no harm.
if it's a potential issue, then we should raise it to the team(maybe ask for a migration before cpou upgrade or just announce it's not supported)

but zzhao is PTO until next Wed, so I did not ask him for a double check yet. And I'm thinking we could get it test once more after the pr merge.

To unblock the pr's merge, my suggestion is to move on, since the test will continue and if it's turned to be an issue, i think we finally will read out a resolution for it, right? because there is still 4.14 and 4.15 sdn customers, so some of them might do cpou upgrade on sdn clusters, then we should cover it. And even finally we come to a conclusion that we should drop this coverage in cpou upgrade, that's acceptable since we cover it and find a known issue on it.